src/Parser/KMLParser.js

```diff
@@ -0,0 +1,26 @@
+import togeojson from '@mapbox/togeojson';
+import GeoJsonParser from 'Parser/GeoJsonParser';
+
+/**
+ * The KMLParser module provides a [parse]{@link module:KMLParser.parse}
+ * method that takes a KML in and gives an object formatted for iTowns
+ * containing all necessary informations to display this KML.
+ *
+ * @module KMLParser
+ */
+export default {
+    /**
+     * Parse a KML file content and return a [FeatureCollection]{@link
+     * module:GeoJsonParser~FeatureCollection}.
+     *
+     * @param {XMLDocument} kml - The KML file content to parse.
+     * @param {Object} options - Options controlling the parsing, see the parse
+     * method of GeoJsonParser for a list of possible options.
```

Comment:
could you define type `jsdoc`, to document this option, (like `cameraTransformOptions`)

Reply:
On it

Reply:
Note: typedef like `cameraTransformOptions` are not yet generated in the documentation, I'm working on it in a separate PR. Meanwhile, I don't think this should stop the PR to be merged ;)